crypto/asn1/standard_methods.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2006-2016 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
I kinda disagree.  The table is copied verbatim from `crypto/asn1/ameth_lib.c`, which has that (c) range, and since the (c) applies on text, not on file name, it should follow.